### PR TITLE
Renames trait datums/character traits to quirks to differentiate them from the other trait system in the code

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
tweak: Character traits are now called character quirks. Functionality remains unaffected.
/:cl:

Had to change a lot of files for this, so sorry if I missed anything. This should help keep future additions cleaner and more differentiated between the two systems. Functionality seems to work okay, but trait setups will be wiped and have to be remade.

### CODE REVIEW

**Comment by XDTM** [comment]
Thank you!

**Comment by Xhuis** [comment]
Thank your MOM

**Comment by iskyp** [comment]
social anxiety and depression are quirky

**Comment by Repukan** [comment]
Port All for One

File: config/game_options.txt [hunk 1]
```diff
@@ -512,7 +512,8 @@ MICE_ROUNDSTART 10
 ## Determines if players are allowed to print integrated circuits, uncomment to allow.
 #IC_PRINTING
 
-## Uncomment to allow roundstart trait selection in the character setup menu.
+## Uncomment to allow roundstart quirk selection in the character setup menu.
+## This used to be named traits, hence the config name, but it handles quirks, not the other kind of trait!
 ROUNDSTART_TRAITS
```
Comment (Cyberboss) [inline config/game_options.txt]: You should rename this here, there's a way to make configs map to other configs for legacy purposes. (I think the FPS config does it)

**Comment by Xhuis** [comment]
Personal stuff came up can't work on this for a while either merge as is or go ahead and close

**Pull request merged**